python/cugraph/cugraph/tests/structure/test_multigraph.py

```diff
@@ -76,7 +76,7 @@ def test_Graph_from_MultiGraph(graph_file):
 
     G = cugraph.Graph(GM)
     Gnx = nx.Graph(GnxM)
-    assert Gnx.number_of_edges() == G.number_of_edges()
+    assert Gnx.number_of_edges() == G.number_of_edges(True)
```

Comment:
Can you add the kwarg name here so it's more obvious?
```suggestion
    assert Gnx.number_of_edges() == G.number_of_edges(directed_edges=True)
```

Reply:
Right